common/GlobalUsings_WinUI.cs

```diff
@@ -7,7 +7,7 @@
 
 global using System.Runtime.InteropServices.WindowsRuntime;
 
-global using CommunityToolkit.Labs.WinUI;
```

Comment:
@Arlodotexe @michael-hawker Should I be touching this? Doesn't compile if not..

Reply:
@Arlodotexe we should remove this as part of #394

Reply:
@michael-hawker Sure, we can try to get both #321 and #387 into #394.

Only note - https://github.com/CommunityToolkit/Labs-Windows/issues/321#issuecomment-1341776315
> We should fix/test https://github.com/CommunityToolkit/Labs-Windows/issues/331 alongside this transition.

We'll do #331 separately another time.

Reply:
Thanks @Arlodotexe, #331 will probably come up a lot for the Toolkit components. If we can agree on a name for the alias (wonder if just `DispatcherQueue` can work?) then would be an easy add if we move the globals, we can investigate if we add that to the PR.

Reply:
Since we're at a critical point for #398 I'd like to close things off quicker than that. There's only pros to closing these off back to back, if we stay on top of it.

Reply:
#394 will have a required fix for `SegmentedControl` not pushing to Nuget ([see logs](https://github.com/CommunityToolkit/Labs-Windows/actions/runs/4377944010/jobs/7662085789#step:12:1292)). This is blocking, so we'll close off #394 once the ticket is filled and just open another for #321.